src/status_im2/subs/communities.cljs

```diff
@@ -235,6 +263,11 @@
                 #(vec (conj %1 %2))
                 {:name             name
                  :emoji            emoji
+                 :gates            (merge
+                                    (when (contains? gates :read)
```

Comment:
it could be just 
```clojure
(-> gate
      (update :read enrich-gates-for-ui)
      (update :write enrich-gates-for-ui))
```
but im not sure why we're doing this in subscription, can this be done when we fetch gates ?

Reply:
At the moment of writing the first subscription, I wasn't sure if we will need the gating data in different formats derived from the basic one. Now it looks like no, so you are right, that can be done when we fetch gates